lms/djangoapps/verify_student/views.py

```diff
@@ -1213,3 +1214,36 @@ def get(self, request):
         """
         IDV_workflow = IDVerificationService.get_verify_location()
         return redirect(IDV_workflow)
+
+
+class PhotoUrlsView(APIView):
+    """
+    This can be used to help debug IDV photos
+    """
+
+    @method_decorator(require_global_staff)
+    def get(self, request):
+        """
+        Endpoint for retrieving photo urls for IDV
+
+        Returns:
+            200 OK
+            {
+                "EdX-ID": receipt_id,
+                "ExpectedName": user profile name,
+                "PhotoID": id photo S3 url,
+                "PhotoIDKey": encrypted photo id key,
+                "UserPhoto": face photo S3 url,
+                "UserPhotoKey": encrypted user photo key,
+            }
+        """
+        receipt_id = request.GET.get('receipt_id')
```

Comment:
Are we getting the `receipt_id` from query string parameter? Why not put that in the URL pattern?

Reply:
I currently have it as a query string parameter, but can put it in the URL path instead